optaplanner-docs/src/main/asciidoc/UseCasesAndExamples/UseCasesAndExamples-chapter.adoc

```diff
@@ -11,9 +11,17 @@
 [[examplesOverview]]
 == Examples Overview
 
-Planner has several examples.
-In this manual we explain mainly using the n queens example and cloud balancing example.
-So it's advisable to read at least those sections.
+Planner has several examples. In this manual we explain mainly using the _n_ queens example and cloud balancing example. So it is advisable to read at least those sections.
```

Comment:
Note (history): The manual is started out focused on the n queens example, but these days: because cloud balancing is the quick start, we should explain every feature mainly with cloud balancing and actively get rid of the use of the n queens example.
The only exception to that is the optimization algorithms themselves, because only with 4 queens it is possible to draw enough solutions states to show the behaviour of the algorithm (see Simulated Annealing algorithm etc)